### PR TITLE
Adds airbrake for error reporting.

### PR DESCRIPTION
**Pull request by bantonj**
Not sure how popular airbrake is, but it's something we use, so thought I'd open this PR in case others might find it useful.

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -18,6 +18,12 @@ func intEnvConfig(i *int, name string) {
 	}
 }
 
+func int64EnvConfig(i *int64, name string) {
+	if env, err := strconv.ParseInt(os.Getenv(name), 10, 64); err == nil {
+		*i = env
+	}
+}
+
```
Comment (DarthSim) [inline config.go:26]: No need for a custom parser for `int64`. You can parse the ID to int and cast it to `int64` here: https://github.com/imgproxy/imgproxy/pull/603/files#diff-695e64e5ace3a24d0e73f02a3555553fa8df9d9045bf759fb729ef870eb62c61R54

File: main.go [hunk 2]
```diff
@@ -107,6 +107,8 @@ func main() {
 		}
 	}
 
+	defer closeErrorsReporting()
+
```
Comment (DarthSim) [inline main.go:111]: Please move this to `run` after `shutdownVips`

**Review by DarthSim — CHANGES_REQUESTED** [review]
Hi Joshua!

Please fix the things I mentioned and I'll merge this.

**Comment by bantonj** [comment]
> Hi Joshua!
> 
> Please fix the things I mentioned and I'll merge this.

Thanks for taking a look. I think I resolved the items you mentioned.

**Comment by DarthSim** [comment]
Thanks for your contribution!

**Pull request merged**